my-app/src/test/java/com/JavaWebServer/app/ServerTest.java

```diff
@@ -35,30 +44,37 @@ public void TestServerTurnsoff () {
   }
 
  @Test 
-  public void  TestServerGetsClientMessage () throws Exception {
+  public void  TestServerRun () throws Exception {
     Thread testThread = new Thread(testServer);
     testThread.start();
     testThread.sleep(100); 
-    
- 
-    Socket clientTestSocket = new Socket("localhost", 8787);
-    System.out.println( "Client: connected to Server");
+
+    java.net.Socket clientTestSocket = new Socket("localhost", 9094);
```

Comment:
You are importing `java.net.Socket` so you don't need to fully qualify the class name.